web/app.js

```diff
@@ -1873,8 +1885,28 @@ function handleMouseWheel(evt) {
   }
 }
 
-window.addEventListener('DOMMouseScroll', handleMouseWheel);
-window.addEventListener('mousewheel', handleMouseWheel);
+//#if !(CHROME || MOZCENTRAL)
+if (window.onwheel === undefined) {
+  // window.onwheel is not set, most likely wheel is not supported.
+  window.addEventListener('DOMMouseScroll', function (evt) {
```

Comment:
#4786 removes support for `DOMMouseScroll` entirely since it's an obsolete Firefox-only event (see e.g. https://developer.mozilla.org/en-US/docs/Web/Events/DOMMouseScroll). What is the reason for keeping it around?